Lesson_1.1.js

```diff
@@ -0,0 +1,15 @@
+function loop(times, callback) {
```

Comment:
```suggestion
function loop(times=0, callback=null) {
```
по условию надо задавать параметрам по умолчанию присваемые значения